### PR TITLE
Automation/new features

### PR DESCRIPTION
**Pull request by larryg01**
refactored code
deleted all unwanted comments in code

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,21 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [5.2.0](https://github.com/klassijs/klassi-js/compare/v5.0.0...v5.2.0) (2023-06-06)
+
+
+### Features
+
+* added new features and refactor s3Upload ([c53736d](https://github.com/klassijs/klassi-js/commit/c53736d5d9ae1c805cc9b985a9eb6657bf762fa5))
```
Comment (carlosbermejop) [inline CHANGELOG.md]: As with the other PR, let's get rid of repetitions on the Changelog

File: runtime/drivers/chromeDriver.js [hunk 2]
```diff
@@ -10,8 +10,7 @@ const program = require('commander');
 const fs = require('fs');
 const path = require('path');
 const { Before } = require('@cucumber/cucumber');
-const { UtamWdioService } = require('wdio-utam-service');
-const utamConfig = require('../utam.config');
+const { filterQuietTags } = require('../.././cucumber.js');
```
Comment (carlosbermejop) [inline runtime/drivers/chromeDriver.js]: Change to 
const { filterQuietTags } = require('../../cucumber.js');

File: runtime/drivers/firefoxDriver.js [hunk 3]
```diff
@@ -8,20 +8,28 @@
 const wdio = require('webdriverio');
 const program = require('commander');
 const { Before } = require('@cucumber/cucumber');
-const { UtamWdioService } = require('wdio-utam-service');
-const utamConfig = require('../utam.config');
+const { filterQuietTags } = require('../.././cucumber.js');
```
Comment (carlosbermejop) [inline runtime/drivers/firefoxDriver.js]: Same as above

File: runtime/drivers/lambdatestDriver.js [hunk 4]
```diff
@@ -7,12 +7,13 @@
...
 const wdio = require('webdriverio');
 const { Before } = require('@cucumber/cucumber');
-const { UtamWdioService } = require('wdio-utam-service');
+// const { UtamWdioService } = require('wdio-utam-service');
 const fs = require('fs-extra');
 const path = require('path');
 const loadConfig = require('../configLoader');
 const lambdatest = require('../remotes/lambdatest');
-const utamConfig = require('../utam.config');
+const { filterQuietTags } = require('../.././cucumber.js');
```
Comment (carlosbermejop) [inline runtime/drivers/lambdatestDriver.js]: Same as above

File: runtime/world.js [hunk 5]
```diff
@@ -5,11 +5,9 @@
  * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
  furnished to do so, subject to the following conditions: The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
  */
-const fs = require('fs-extra');
-const merge = require('merge');
-const requireDir = require('require-dir');
+const { setDefaultTimeout, Before } = require('@cucumber/cucumber');
 const getRemote = require('./getRemote');
-// const helpers = require('./helpers');
+const { filterQuietTags } = require('.././cucumber.js');
```
Comment (carlosbermejop) [inline runtime/world.js]: This should be either require('../../cucumber.js'); or require('../cucumber.js');

**Review by carlosbermejop — CHANGES_REQUESTED** [review]
Some changes required

File: CHANGELOG.md (hunk 1 above)
Comment (larryg01) [reply CHANGELOG.md]: removed

File: runtime/drivers/lambdatestDriver.js (hunk 4 above)
Comment (larryg01) [reply runtime/drivers/lambdatestDriver.js]: done

File: runtime/drivers/chromeDriver.js (hunk 2 above)
Comment (larryg01) [reply runtime/drivers/chromeDriver.js]: done

File: runtime/world.js (hunk 5 above)
Comment (larryg01) [reply runtime/world.js]: Done its '../'

**Pull request merged**